web-common/src/features/dashboards/dashboard-stores.ts

```diff
@@ -302,3 +302,5 @@ export function useDashboardStore(
 }
 
 export const calendlyModalStore: Writable<string> = writable("");
```

Comment:
Also do we still need Calendly components and store now in the codebase? If not, we can remove those as well.